addons/payment_transfer/models/payment.py

```diff
@@ -18,35 +18,6 @@ class TransferPaymentAcquirer(models.Model):
     def transfer_get_form_action_url(self):
         return '/payment/transfer/feedback'
 
-    def _format_transfer_data(self):
```

Comment:
An improvement could be to do an onchange on the payment type that sets a default value, but it's not clean neither because you could have strange effect if the user sets Wire Transfer, then Credit Card. (the value will be wrong for credit cards processing)

Even if we decide to do something to improve the default value of bank wire transfers, we can view that as a new feature, in a separate branch.